repose-aggregator/components/filters/uri-stripper-filter/build.gradle

```diff
@@ -22,4 +22,5 @@ dependencies {
     testCompile "org.mockito:mockito-all"
     testCompile "org.springframework:spring-test"
     testCompile "org.slf4j:jcl-over-slf4j"
+    testCompile "xerces:xerces-xsd11"
```

Comment:
Where did this come from?

Reply:
Another dependency is bringing an older version of Xerces in transitively and was causing a `java.lang.NoSuchMethodError`.